### DISCUSSION TITLE
Write permission

### DISCUSSION THREAD

**Discussion by dangerbaby** [coastal-hazards-system/StormSim-Library#27] [General]
Hey SS Gang,

I  worked on getting my download of SS working this week.  Much of the time was tracking down issues that were on my end--long story.  I have made some changes to my new branch, though, that should allow uptake of info from external files in a more general way--meaning that it's not dependent on  Windows ( as readcell.m is).  I tried to push brads_branch to the hub, but I don't think that I have write permission.  Perhaps we could alter that permission setting.  

Brad

**Comment by fgarcia0128** [comment]
Can you do a fork with your current permissions?

**Reply by dangerbaby** [reply]
Perhaps I could fork, but then I would lack the ability to push back to the repo, which is required if you want my changes.  I should mention, however,  that my use of git is pretty inept.

**Reply by fgarcia0128** [reply]
You will be able to push up to the repo by submitting a pull request. 

Steps:
1- Fork StormSim Library
2-Go to your fork repo home page 
3-Open Pull request tab and Create a Pull Request 

Pull request is the best way for any collaborator to submit changes for review before merging with the master branch.

**Reply by dangerbaby** [reply]
Am I listed as a collaborator in Settings?

**Reply by fgarcia0128** [reply]
Yes

**Reply by dangerbaby** [reply]
Seems like forking is disabled
![PXL_20230518_172324443](https://github.com/Coastal-Hazards-System/StormSim-Library/assets/3789527/fdc08446-65bd-4f9d-8fe8-b450545dc91d)

**Comment by fgarcia0128** [comment]
Should be fixed
